.drone.yml

```diff
@@ -107,6 +108,19 @@ steps:
   depends_on:
   - build-backend
 
+- name: ensure-cuetsified
+  image: grafana/build-container:1.4.3
+  commands:
+  - ./bin/linux-amd64/grafana-cli cue gen-ts --grafana-root .
+  - "# The above command generates Typescript files (*.gen.ts) from all appropriate .cue files."
+  - "# It is required that the generated Typescript be in sync with the input CUE files."
+  - "# ...Modulo eslint auto-fixes...:"
+  - ./node_modules/.bin/eslint . --ext .gen.ts --fix
```

Comment:
Is this an acceptable way to invoke our linter? Seems to be what we do with husky in `package.json`